client/src/test/java/org/apache/rocketmq/client/consumer/rebalance/AllocateMessageQueueConsitentHashTest.java

```diff
@@ -92,9 +92,9 @@ public void testAllocate2() {
 
     @Test
     public void testRun100RandomCase() {
-        for (int i = 0; i < 100; i++) {
-            int consumerSize = new Random().nextInt(200) + 1;//1-200
-            int queueSize = new Random().nextInt(100) + 1;//1-100
+        for (int i = 0; i < 10; i++) {
+            int consumerSize = new Random().nextInt(20) + 1;//1-20
```

Comment:
Guys, what's the point to reduce these numbers (in other tests in this PR too)?
@vongosling @zhouxinyu 

IMO, we need to have a pretty high number for messages/queues/etc. in unit tests, because it is a large-scale messaging system. Unless it makes tests really slow.